Mathlib/Logic/Lemmas.lean

```diff
@@ -55,3 +55,12 @@ theorem ite_ite_distrib_left : ite p a (ite q b c) = ite q (ite p a b) (ite p a
 
 theorem ite_ite_distrib_right : ite p (ite q a b) c = ite q (ite p a c) (ite p b c) :=
   dite_dite_distrib_right
+
+set_option autoImplicit false
```

Comment:
I guess we don't want to keep this line.

Reply:
Whoops